act-framework.bs

```diff
@@ -42,10 +42,8 @@ Rule Description {#structure-description}
 
 Each ACT Rule MUST have a description that:
 
-* Is in plain language;
-* Explains the accessibility requirement being tested;
-* Lists any limitations and assumptions for the test;
-* Explains the failure conditions and their reasons.
+* Is in plain language and provides a brief description of what the rule does.
+* Explains the accessibility requirement being tested such as the WCAG SC or the technique the rule maps to; For example WCAG 2.0 Technique H67.
```

Comment:
If I recall correctly, a rule always applies to at least one success criterion, and may map to any number of techniques? If so, should this be separated into two points:
 - Lists the accessibility requirement being tested, such as the WCAG Success Criterion
 - Refers to any related information, such as the WCAG Techniques it maps to